src/routes/(root)/api/rest-v1/[...slug].tsx

```diff
@@ -54,14 +54,9 @@ export default function ApiV1Docs() {
           비인증 결제, 정기 자동결제 등 부가기능을 위한 REST API도 제공합니다.
         </prose.p>
         <Hint style="danger">
-          2024년 9월 1일부로 포트원 V1 API에 대해 일부 보안 규격이 지원
-          종료됩니다.
-          <br />
-          자세한 사항은
-          <a href="/opi/ko/support/tls-support?v=v1" style="color:orange">
-            TLS 지원 범위
-          </a>
-          를 참고해주세요.
+          2024년 9월 1일부로 포트원 V1 API에 대해 일부 보안 규격이 지원 종료됩니다.
+      
+          자세한 사항은 [TLS 지원 범위](/opi/ko/support/tls-support?v=v1)를 참고해주세요.
```

Comment:
```suggestion
          <prose.p>
            2024년 9월 1일부로 포트원 V1 API에 대해 일부 보안 규격이 지원
            종료됩니다.
          </prose.p>
          <prose.p>
            자세한 사항은
            <a href="/opi/ko/support/tls-support?v=v1" style="color:orange">
              TLS 지원 범위
            </a>
            를 참고해주세요.
          </prose.p>
```

TSX 파일에서는 MDX 문법을 사용할 수 없기 때문에 `<prose.p>` 컴포넌트를 사용해서 MDX 스타일을 적용해야 합니다.

Preview:
![image](https://github.com/user-attachments/assets/fca95f01-eb95-4f79-a949-e3e1a8d7940b)

Reply:
수정 했습니다. 다만 color: orange;를 사용 시 메인페이지나 다른 페이지와 color가 다르게 들어가다보니, className을 직접 넣었습니다. Deploy 되면, 다시 한번 확인해보려고 합니다.